libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -43,7 +43,7 @@ const AP_Param::GroupInfo AP_MotorsHeli_Dual::var_info[] = {
 
     // @Param: DCP_YAW
     // @DisplayName: Differential-Collective-Pitch Yaw Mixing
-    // @Description: Feed-forward compensation to automatically add yaw input when differential collective pitch is applied.
+    // @Description: Feed-forward compensation to automatically add yaw input when differential collective pitch is applied. Yaw revereser neutral width for intermeshing mode, set to -1 to disable reverser
```

Comment:
If

Reply:
If you look at the math formula, increasing DCP_YAW will shrink the region that yaw output is reduced. 0 means yaw output is always 0 and infinity makes reversal instantaneous. (See the graph in another comment)

Even though 0 also disable it, I was taught to give it some safety range to avoid floating point accuracy error. And also because some other int based parameters also use -1 to disable.

Another possible issue with 0 is `0.00001` may be display rounded as 0 and saying `-1` is better than saying "any negative number" in my opinion.

Reply:
> why not tell the user that 0 disables the feature just like it does for tandems and side-by-side configurations?

Because disabled behavior and behavior at 0.00001 of the reverser is vastly different, unlike that of the other two.

Reply:
Ok now I know why I don’t care for this implementation.  The parameter you are using is specifically for coupling due to differential collective  inputs. Yours is an issue due to collective going negative and causing control issues in the yaw axis.  So if we add this feature, we need a new parameter which is what you had in your original implementation.  You called it REVP, I think is what I saw.  Not sure if that is the right name.

Reply:
I got no comment when I asked whether reusing parameter will cause confusion. Apparently it did.

This guide for main code suggests there is a limit to parameter list but I couldn't find what the limit is and there's no empty slot.
httpss://ardupilot.org/dev/docs/code-overview-adding-a-new-parameter.html
Will simply extending `AP_GROUPINFO("YAW_REVP", 22...` at the end of list work?

Reply:
Pitt,
    I apologize that I missed that question and if you thought my previous response was rude.

  As far as the limit to parameter list, this parameter is being added in a library so you wouldn't have to find an open slot.  You would just add it to the end of the list of the parameters.  So this would be 22 as you have indicated.  I would like you to name the parameter "YAW_REV_EXPO".  Please be sure that the description states that it is for intermeshing only and I am fine with this parameter being disabled with "-1".